mindsdb/integrations/handlers/altibase_handler/README.md

```diff
@@ -33,6 +36,28 @@ parameters={
 };
 ~~~~
 
+~~~~sql
+CREATE DATABASE altibase_datasource
+WITH
+engine='Altibase',
```

Comment:
Engine name is case sensitive, so for now, we can keep the `altibase` as set in the handler https://github.com/mindsdb/mindsdb/pull/9159/files#diff-c8ab3928263afa27fed126cbf44f81718ca80b6d440cd190a489c423319bcf0fR28

Reply:
Hi. I copied that from 29 lines. I agree with you that use 'altibase'
So you mean I need the pull-request again?
I'm first pull-request in my life.